packages/configure/src/utils/authenticated.client.ts

```diff
@@ -2,8 +2,7 @@ import { Configuration, DefaultApi } from '@flatfile/api'
 // TODO: We will need to make this conditional depending on if it's in the NodeVM or the Browser
 import fetch, { RequestInit } from 'node-fetch'
 
-const FLATFILE_API_URL =
-  process.env.AGENT_INTERNAL_URL || 'http://localhost:3000'
+const FLATFILE_API_URL = process.env.AGENT_INTERNAL_URL
```

Comment:
How about defaulting to platform and skip the `null` checks below?

```suggestion
const FLATFILE_API_URL = process.env.AGENT_INTERNAL_URL || 'https://platform.flatfile.com/api'
```